kubernetes/zenko/charts/zenko-queue-manager/requirements.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+- name: zookeeper
```

Comment:
1. Yeah, Kafka manager only needs Zookeeper to initialize properly (to manage state). This can either be its personal quorum or a shared one with a Kafka cluster.

2. Correct.

The reason for this dependency is only to match the chart I'm pushing upstream since that one won't pass CI without stabilizing and it needs Zookeeper for that (readiness probe).